install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -138,19 +138,40 @@ variables begin with `openshift_storage_glusterfs_registry_`. A few other
 variables, such as `openshift_hosted_registry_storage_kind`, interact with the
 GlusterFS clusters.
 
-It is recommended to specify version tags for all containerized components.
-This is primarily to prevent components, particularly the {gluster} pods, from
-upgrading after an outage which may lead to a cluster of widely disparate
-software versions. The relevant variables are:
-
-- `openshift_storage_glusterfs_version`
-- `openshift_storage_glusterfs_block_version`
-- `openshift_storage_glusterfs_s3_version`
-- `openshift_storage_glusterfs_heketi_version`
-- `openshift_storage_glusterfs_registry_version`
-- `openshift_storage_glusterfs_registry_block_version`
-- `openshift_storage_glusterfs_registry_s3_version`
-- `openshift_storage_glusterfs_registry_heketi_version`
+It is recommended to specify the image name and version tags for all
+containerized components. This is primarily to prevent components, particularly
+the {gluster} pods, from upgrading after an outage which may lead to a cluster
+of widely disparate software versions. The relevant variables are:
+
+- `openshift_storage_glusterfs_image`
+- `openshift_storage_glusterfs_block_image`
+- `openshift_storage_glusterfs_s3_image`
+- `openshift_storage_glusterfs_heketi_image`
+- `openshift_storage_glusterfs_registry_image`
+- `openshift_storage_glusterfs_registry_block_image`
+- `openshift_storage_glusterfs_registry_s3_image`
+- `openshift_storage_glusterfs_registry_heketi_image`
+
+[NOTE]
+====
+The image variables for gluster-block and gluster-s3 are only necessary if the
+corresponding deployment variables (the variables ending in `_block_deploy` and
+`_s3_deploy`, respectively) are true.
+====
+
+ifdef::openshift-enterprise[]
+The following are the recommended values for this release of {product-title}:
+
+- `openshift_storage_glusterfs_image=registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.10`
+- `openshift_storage_glusterfs_block_image=registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:v3.10`
+- `openshift_storage_glusterfs_s3_image=registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:v3.10`
+- `openshift_storage_glusterfs_heketi_image=registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:v3.10`
+
+[NOTE]
+====
+The values are the same for the corresponding `_registry_` variables.
```

Comment:
Remove extra 'the' s/The values are the same for .../The values are same for ...